ui/packages/catalog-ui-search/src/main/webapp/react-component/container/sharing/sharing.tsx

```diff
@@ -229,6 +233,7 @@ export class Sharing extends React.Component<Props, State> {
   }
 
   remove = (i: number) => {
+    this.usersToUnsubscribe.push(this.state.items[i].value)
```

Comment:
not sure if `remove` is triggered if the user's permissions are set to `None` in the dropdown